src/main/java/com/github/wovnio/wovnjava/HtmlReplaceMarker.java

```diff
@@ -0,0 +1,39 @@
+package com.github.wovnio.wovnjava;
+
+import java.util.HashMap;
+
+class HtmlReplaceMarker {
+    private final String WOVN_MARKER_PREFIX = "wovn-marker-";
+    private final HashMap<String, String> mappedValues;
```

Comment:
I'm not familiar with Java.
Is `final` correct in this case?

Reply:
`final` mean the reference to `mappedValues` cannot change. But the contents can change, as long as it's the same HashMap object. So it is ok.

Reply:
The `final` for the `String` means `WOVN_MARKER_PREFIX` is a constant, but that is because Strings are immutable in Java. When you try to change a String you must create a new one and therefore the reference has to change.

Reply:
This is a best practice in java, similar to `readonly` in C# for example.